web/themes/custom/westy/templates/block/block--views-block--alert-block-alert.html.twig

```diff
@@ -0,0 +1,10 @@
+{#
+/**
+ * @file
+ * Template for a block.
```

Comment:
Please add a note in the comments that this template is to override the utility class applied via block--views-block.html.twig.